cli/src/main/scala/mdoc/internal/cli/CliEnrichments.scala

```diff
@@ -1,20 +1,20 @@
 package mdoc.internal.cli
 
+import coursierapi.Dependency
+
+import java.io.PrintStream
 import java.nio.charset.StandardCharsets
 import java.nio.file.Files
 import java.nio.file.Paths
 import java.nio.file.StandardCopyOption
 import scala.meta.Input
 import scala.meta.Position
+import scala.meta.internal.io.FileIO
+import scala.meta.internal.io.PathIO
 import scala.meta.io.AbsolutePath
 import scala.meta.io.RelativePath
-import scala.meta.internal.io.PathIO
-import scala.util.control.NonFatal
-import coursierapi.Dependency
-import scala.meta.internal.io.FileIO
-import scala.meta.internal.inputs._
```

Comment:
avoid binary-breaking change in https://github.com/scalameta/scalameta/pull/2812/files#diff-28130ee0df4fbb1f48a55962940f2c12bbb8ea375ccffcc16435ad54d1f0b9b7R33 :+1: